frontend/packages/console-app/locales/zh/nodes.json

```diff
@@ -0,0 +1,82 @@
+{
+  "This action cannot be undone. Deleting a node will instruct Kubernetes that the node is down or unrecoverable and delete all pods scheduled to that node. If the node is still running but unresponsive and the node is deleted, stateful workloads and persistent volumes may suffer corruption or data loss. Only delete a node that you have confirmed is completely stopped and cannot be restored.": "此操作无法撤消。删除节点会指示 Kubernetes 使该节点停止或无法恢复，并删除所有调度到该节点的 pod。如果节点仍然在运行但无法响应，则该节点被删除，有状态的工作负载和持久性卷可能会崩溃或造成数据丢失。只有删除已确认的节点才会完全停止且无法恢复。",
+  "Delete node": "删除节点",
+  "Mark as unschedulable": "将节点标记为不可调度。",
+  "Unschedulable nodes won't accept new pods. This is useful for scheduling maintenance or preparing to decommission a node.": "不可调度的节点不会接受新的 pod。这在调度维护或准备退出节点时很有用。",
+  "Mark unschedulable": "标记为不可调度。",
+  "Activity": "活跃",
+  "View events": "查看事件",
+  "Details": "详情",
+  "View all": "查看所有",
+  "Node name": "节点名称",
+  "Role": "角色",
+  "Instance type": "实例类型",
+  "Zone": "区",
+  "Node addresses": "节点地址",
+  "Inventory": "库存",
+  "Image": "镜像",
+  "Images": "镜像",
+  "Not available": "不可用",
+  "See breakdown": "查看分解",
+  "See details": "查看详情",
+  "Health checks": "健康检查",
+  "{{ cpuMessage }}": "{{ cpuMessage }}",
+  "{{ memoryMessage }}": "{{ memoryMessage }}",
+  "{{ machineHealthCheckLabelPlural }} automatically remediate node health issues.": "{{ machineHealthCheckLabelPlural }} 自动修复节点的健康状况。",
+  "Conditions": "条件",
+  "Status": "状态",
+  "Reboot pending": "等待重启",
+  "Reprovision pending": "等待重新置备",
+  "Only one {{ machineHealthCheckLabel }} resource should match this node.": "只有一个 {{ machineHealthCheckLabel }} 资源应该与此节点匹配。",
+  "Not configured": "未配置",
+  "CPU": "CPU",
+  "Memory": "内存",
+  "Filesystem": "Filesystem",
+  "Network in": "网络入",
+  "Network out": "网络出",
+  "Utilization": "使用率",
+  "Network transfer": "网络传输",
+  "Pod count": "Pod 数量",
+  "Node conditions": "节点条件",
+  "Type": "类型",
+  "Reason": "原因",
+  "Updated": "已更新",
+  "Changed": "已修改",
+  "Name": "名称",
+  "Size": "Size",
```

Comment:
"Size" needs to be translated into Chinese.

Reply:
This has not been addressed.

Reply:
nodes is not received in our batch for sprint 192 so will probably need to be sent again to address this.

Reply:
I'll send over the full batch today closer to EOB.